src/css/object/project/keyvisual.scss

```
@@ -1,2 +1,52 @@
 @use '../../object/utility/utility.scss';
+@use '../../foundation/variables';
 // ページ上部のキービジュアルに関するスタイル
+
+.p-key {
```

[nits]

以下が適切かと思います！

```suggestion
.p-keyvisual {
```

---

承知しました！修正しました！